test/10-api.js

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright (c) 2020-2021 Digital Bazaar, Inc. All rights reserved.
+ */
+
+describe('api', () => {
+  it('should work');
```

Comment:
Remove this, it inspires false confidence in proper workingness. ;-)

See test comment above for possible way to have real(er) tests here.